server/src/service/session.rs

```diff
@@ -0,0 +1,37 @@
+// org.freedesktop.Secret.Session
+
+use serde::{Deserialize, Serialize};
+use zbus::{dbus_interface, fdo, zvariant};
+use zvariant::{ObjectPath, OwnedObjectPath, Type};
+
+#[derive(Debug, Serialize, Deserialize, Type)]
```

Comment:
Should it implement Serialize/Desrialize/Type?

Reply:
Yeah! Removed.